Assets/MixedRealityToolkit/Utilities/Editor/DependencyWindow.cs

```diff
@@ -0,0 +1,339 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+using Object = UnityEngine.Object;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
+{
+    public class DependencyWindow : EditorWindow
+    {
+        private Object assetSelection = null;
```

Comment:
Have you tried showing and then close tab the window? Do all variables reset to defaults?

Reply:
When you close the window it destroys the DependencyWindow and hence the dependencyGraph, so everything is restored to the defaults when you re-open the window.